src/js/components/FormField/FormField.js

```diff
@@ -45,10 +45,41 @@ const FormFieldContentBox = styled(Box)`
   ${props => props.focus && focusStyle({ justBorder: true })}
 `;
 
-const Message = ({ message, ...rest }) => {
+const StyledMessageContainer = styled(Box)`
+  ${props =>
+    props.messageType &&
+    props.theme.formField[props.messageType].container &&
+    props.theme.formField[props.messageType].container.extend}
+`;
+
+const Message = ({ error, info, message, ...rest }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+
+  let messageType;
+  if (error) messageType = 'error';
+  if (info) messageType = 'info';
+  const icon =
+    theme.formField[messageType] && theme.formField[messageType].icon;
+
+  const containerProps =
+    theme.formField[messageType] && theme.formField[messageType].container;
+
   if (message) {
-    if (typeof message === 'string') return <Text {...rest}>{message}</Text>;
-    return <Box {...rest}>{message}</Box>;
+    let messageContent;
+    if (typeof message === 'string')
+      messageContent = <Text {...rest}>{message}</Text>;
+    else messageContent = <Box {...rest}>{message}</Box>;
+
+    return (
+      <StyledMessageContainer
+        direction={icon ? 'row' : 'column'}
```

Comment:
Why not always `direction="row"`?

Reply:
You're right, I think always having row is fine.